influxdb/content.md

```diff
@@ -1,643 +1,221 @@
-    -v influxdb2-config:/etc/influxdb2 \
-    -v "$PWD/influxdb.conf:/root/influxdb/influxdb.conf" \
-    -e DOCKER_INFLUXDB_INIT_MODE=upgrade \
-    -e DOCKER_INFLUXDB_INIT_USERNAME=my-user \
-    -e DOCKER_INFLUXDB_INIT_PASSWORD=my-password \
-    -e DOCKER_INFLUXDB_INIT_ORG=my-org \
-    -e DOCKER_INFLUXDB_INIT_BUCKET=my-bucket \
-    -e DOCKER_INFLUXDB_INIT_UPGRADE_V1_CONFIG=/root/influxdb/influxdb.conf \
-    %%IMAGE%%:2
-```
+%%COMPOSE%%
 
-To use your custom paths instead of InfluxDB v2 default paths, run the following command:
+Start your container:
 
 ```bash
-docker run -p 8086:8086 \
-    -v influxdb:/root/influxdb/data \
-    -v influxdb2:/root/influxdb2/data \
-    -v influxdb2-config:/etc/influxdb2 \
-    -v "$PWD/influxdb.conf:/root/influxdb/influxdb.conf" \
-    -e DOCKER_INFLUXDB_INIT_MODE=upgrade \
-    -e DOCKER_INFLUXDB_INIT_USERNAME=my-user \
-    -e DOCKER_INFLUXDB_INIT_PASSWORD=my-password \
-    -e DOCKER_INFLUXDB_INIT_ORG=my-org \
-    -e DOCKER_INFLUXDB_INIT_BUCKET=my-bucket \
-    -e DOCKER_INFLUXDB_INIT_UPGRADE_V1_CONFIG=/root/influxdb/influxdb.conf \
-    -e DOCKER_INFLUXDB_CONFIG_PATH=/root/influxdb2/config.toml \
-    -e DOCKER_INFLUXDB_BOLT_PATH=/root/influxdb2/influxdb.bolt \
-    -e DOCKER_INFLUXDB_ENGINE_PATH=/root/influxdb2/engine \
-    %%IMAGE%%:2
+docker compose pull && docker compose run influxdb3-enterprise
 ```
 
-To learn more about the upgrade process, see the [v1-to-v2 upgrade guide](https://docs.influxdata.com/influxdb/v2.0/upgrade/v1-to-v2/).
-
-### Upgrading from quay.io-hosted InfluxDB 2.x image
-
-Early Docker builds of InfluxDB 2.x were hosted at `quay.io/influxdb/influxdb` and contained the `influx` and `influxd` binaries without any default configuration or helper scripts. By default, the `influxd` process stored data in `/root/.influxdbv2`.
-
-Starting with `v2.0.4`, we restored the InfluxDB Docker Hub build, which defaults to storing data in `/var/lib/influxdb2`. If you upgrade directly from `quay.io/influxdb/influxdb` to `influxdb:2.0.4` using the default settings, InfluxDB won't be able to find your existing data files.
-
-To avoid this problem when migrating from `quay.io/influxdb/influxdb` to `influxdb:2.0`, choose one of the following:
-
-#### Update the mount to use the InfluxDB default
+## Docker with mounted file style object store
 
-To use the InfluxDB Docker Hub data path, start a container that mounts your data volume into `/var/lib/influxdb2`--for example, if you used the following command to start the InfluxDB quay.io container:
+Run this command to start the InfluxDB 3 Enterprise container
 
 ```bash
-# quay.io InfluxDB 2.x container 
-docker run -p 8086:8086 \
-    -v "$PWD:/root/.influxdbv2" \
-    quay.io/influxdb/influxdb:v2.0.3
+docker run -d --name influxdb3-enterprise -p 8086:8086 \
+  -v $PWD/plugins:/plugins \
+  -v $PWD/data:/var/lib/influxdb3 \
+  -e INFLUX_LICENSE_KEY="YOUR_LICENSE_KEY" \
+  influxdb:enterprise serve \
+    --cluster-id cluster1 \
```

Comment:
We should have an example using available environment variables, specific for Docker startup (if any) and InfluxDB3 variables. Users will want to source them from a file (`.env`, for example)